src/main/core/Event/Functional/ToolOpenEvent.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Claroline Connect package.
+ *
+ * (c) Claroline Consortium <consortium@claroline.net>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Claroline\CoreBundle\Event\Functional;
+
+use Claroline\CoreBundle\Entity\User;
+use Claroline\CoreBundle\Entity\Workspace\Workspace;
+use Symfony\Contracts\EventDispatcher\Event;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class ToolOpenEvent extends Event
```

Comment:
An event already exist : `Claroline\CoreBundle\Event\Tool\OpenToolEvent`. Must be removed.

Reply:
Done, thx :)